app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminCategoriesClickDoneButtonOnPopupActionGroup.xml

```diff
@@ -8,11 +8,12 @@
 
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
               xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
-    <actionGroup name="AdminSubmitCategoriesPopupActionGroup">
+    <actionGroup name="AdminCategoriesClickDoneButtonOnPopupActionGroup">
```

Comment:
I would not recommend you renaming this action group without strong necessity. It's a backward-incompatible change since 3rd party modules may rely on this core action group.

Reply:
**Done it** 
Also I see that failed tests are not related with my changes. 
Thanks.